code_map.py

```diff
@@ -886,6 +888,10 @@ def on_close(self, view):
                 reset_layout()
             sublime.set_timeout_async(focus_source_code)
 
+            # try to reactivate, in case another window with Codemap is open,
+            # or a new workspace has been loaded
+            sublime.set_timeout_async(lambda: reactivate())
+
```

Comment:
I moved reactivation check in the on_close listener because it was still giving problems.